examples/api/test/material/data_table/data_table.1_test.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_api_samples/material/data_table/data_table.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('DataTable is scrollable', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.DataTableExampleApp(),
+    );
+
+    expect(find.text('Row 0').hitTestable(), findsOneWidget);
+    expect(find.text('Row 19').hitTestable(), findsNothing);
+
+    await tester.ensureVisible(find.text('Row 19'));
+
+    expect(find.text('Row 0').hitTestable(), findsNothing);
+    expect(find.text('Row 19').hitTestable(), findsOneWidget);
```

Comment:
```suggestion
    expect(find.byType(SingleChildScrollView), findsOneWidget);

    expect(tester.getTopLeft(find.text('Row 5')), const Offset(66.0, 366.0));

    await tester.drag(find.byType(SingleChildScrollView), const Offset(0.0, -200.0));
    await tester.pumpAndSettle();

    expect(tester.getTopLeft(find.text('Row 5')), const Offset(66.0, 186.0));
```

Reply:
@Chinmay-KB 
I have updated the test to test `SingleChildScrollView` scrolls the `DataTable`.

The previous test didn't exactly test the changes made to the code sample.